environment.yml

```diff
@@ -15,6 +15,11 @@ dependencies:
 
   - pip
   - pip:
+    - torch 
+    - torchvision
```

Comment:
see this file https://github.com/ramp-kits/stroke/blob/master/environment.yml

torch with gpu is better to be installed with conda not pip

Reply:
thank you, I'm fixing it